_includes/frontpage/docs.html

```diff
@@ -1,39 +1,59 @@
 <section id="docs">
   <div class="container">
-    <h2>Documentation
-        <img class="shield" alt="GitHub package.json version"
+    <h2>Getting Started
+    <a href="https://github.com/MathJax/MathJax" class="shield">
+      <img class="shield" alt="MathJax version on GitHub"
            src="https://img.shields.io/github/package-json/v/mathjax/MathJax">
-       <img class="shield" alt="GitHub package.json version (branch)"
+      </img>
+    </a>
+    <a href="https://github.com/MathJax/MathJax/tree/legacy-v2" class="shield">
+      <img class="shield" alt="MathJax version 2 legacy version on GitHub"
            src="https://img.shields.io/github/package-json/v/mathjax/MathJax/legacy-v2">
+      </img>
+    </a>
+    <a href="https://www.npmjs.com/package/mathjax" class="shield">
+      <img class="shield" alt="MathJax npm package"
+           src="https://img.shields.io/npm/v/mathjax">
+      </img>
+    <a href="https://www.npmjs.com/package/mathjax-full" class="shield">
+      <img class="shield" alt="MathJax sources npm package"
+           src="https://img.shields.io/npm/v/mathjax-full?label=npm@sources">
+      </img>
+    </a>
+    <a href="https://www.npmjs.com/package/mathjax-node" class="shield">
+      <img class="shield" alt="MathJax version 2 legacy npm package"
+           src="https://img.shields.io/npm/v/mathjax-node?label=npm@legacy-v2">
+      </img>
+    </a>
     </h2>
 
     <div class="cards">
 
-      <div class="card">
+      <div class="card" id="to-gettingstarted">
         <div>
-          <h3>Getting Started</h3>
+          <h3>Web Integration</h3>
           <p>
             <img src="img/code.svg" alt="" class="card--img">
             Whether you're a casual user, a serious author, or a professional developer, it's easy to
             integrate MathJax.
           </p>
         </div>
-        <a class="btn" href="#gettingstarted" id="to-gettingstarted">Learn more</a>
+        <a class="btn" href="#gettingstarted">Start now</a>
```

Comment:
I agree that the bounce-back is not great, but if you are going to make a change like this, it should be done to all the buttons in all the section, not just this one.  

It turns out that there is a script that modifies the targets of the back links and it is controlled by an array in the `main.js` file:

https://github.com/mathjax/MathJax-website/blob/f2f8fa592a41e8ae09f1342ffb0559573c6fff93/js/main.js#L67-L100

You need to add `#to-intsallnow` to that list to get consistent behavior for the new card.

If you want to change the bound-back to not use this approach (it was not my doing), and change all the links, I'm good with that, but the current setup is inconsistent from section to section, and for this new card compared to the other cards.

Reply:
I've updated the table so all cards have the same original behaviour.